src/test/java/com/github/tomakehurst/wiremock/extension/ExtensionLifeCycleAcceptanceTest.java

```diff
@@ -65,8 +66,8 @@ public void reset() {
     }
 
     @Override
-    public void info(String message) {
```

Comment:
Let's keep this method for backwards compatibility, with a default impl delegating to the new supplier-based one.